files/SID/04-VTSP

```diff
@@ -0,0 +1,66 @@
+VTSP-09 BORNA1A           SP001 BORNA
```

Comment:
https://www.ivao.aero/softdev/IvAc/sectorspec.htm#star

คือไม่แน่ใจว่า spacebar มันสำคัญรึเปล่า?

Reply:
![image](https://user-images.githubusercontent.com/3850197/33807135-b55fa45c-de04-11e7-917f-458d833b0f7b.png)

ตัวอย่างจาก website IVAO

Reply:
Each line contains 85 characters, each field separated by 1 space:

- Name of SID: 25 characters, pad with spaces
- Starting Latitude: 14 characters, pad with zeros
- Starting Longitude: 14 characters, pad with zeros
- Ending Latitude: 14 characters, pad with zeros
- Ending Longitude: 14 characters, pad with zeros

Use VOR, NDB, Airport or Fixes names in place of Latitude/Longitudes in starting and ending co-ordinates. If you specify a VOR, NDB, Airport or FIX for latitude and you leave longitude blank then the one of the latitude will be used.

Name is only needed once, lines below that first line belong to that SID, until new name specified.